pubtools/_quay/signature_handler.py

```diff
@@ -410,6 +432,20 @@ def validate_radas_messages(self, claim_messages, signature_messages):
                 )
             )
 
+    def remove_outdated_signatures(self, signatures_to_remove):
```

Comment:
Is this method even necessary?

Reply:
yes, I guess it doesn't make sense. There won't be any extra logic needed for this in the future